collector/cpu.go

```diff
@@ -116,99 +152,174 @@ type Win32_PerfRawData_Counters_ProcessorInformation struct {
 	PerformanceLimitFlags       uint64
 	ProcessorFrequency          uint64
 	ProcessorStateFlags         uint64
-}*/
+}
 
 func (c *CPUCollector) collect(ch chan<- prometheus.Metric) (*prometheus.Desc, error) {
-	var dst []Win32_PerfRawData_PerfOS_Processor
-	q := queryAll(&dst)
-	if err := wmi.Query(q, &dst); err != nil {
-		return nil, err
-	}
+	version := getWindowsVersion()
+	// Windows version by number https://docs.microsoft.com/en-us/windows/desktop/sysinfo/operating-system-version
+	if version > 6.05 {
```

Comment:
Add a comment here explaining why 6.05 is the magic version :)

Reply:
Hi, since `Get-WmiObject Win32_PerfRawData_PerfOS_Processor` work on Windows before 2008 and earlier ( Windows version 6.0 or less) , and `Get-WmiObject Win32_PerfRawData_Counters_ProcessorInformation` on 2008 R2 +(Windows 6.1 or higher) , the split number is 6.05. I will add comment about it.

Reply:
Yeah, I guessed so too, but I think it makes sense to spell it out :)